.github/workflows/atlantis-image.yml

```diff
@@ -51,6 +52,7 @@ jobs:
         images: |
           atlantis
         labels: |
+          authors="Anubhav Mishra, Luke Kysow"
```

Comment:
Let's use the standardized labels

ref: https://github.com/opencontainers/image-spec/blob/main/annotations.md

```suggestion
          org.opencontainers.image.authors="Anubhav Mishra, Luke Kysow"
```

Reply:
We also may want to update the list of authors here to ones who will be maintaining the image. Anubhav hasn't committed in a long time and Luke isn't part of the day to day of the container maintenance.

Honestly, it may be better to omit the label entirely.

Reply:
Is there a group email that we can reference instead of individuals?

Reply:
There isn't but perhaps there should be?

Reply:
@nitrocode I'll merge your commit suggestion, but maybe we later update it to be something more agnostic like "Atlantis Maintainers"